packages/core/src/sass/app/_common/components/wallet.scss

```diff
@@ -1,63 +1,812 @@
+                        100% 4130.74% at 0% 100%,
+                        rgba(255, 100, 68, 0.4) 0%,
+                        rgba(255, 100, 68, 0.16) 50.52%,
+                        rgba(255, 68, 79, 0.4) 100%
+                    )
+                    #212329,
+                'desktop':
+                    radial-gradient(
+                        100% 4130.74% at 0% 100%,
+                        rgba(255, 100, 68, 0.4) 0%,
+                        rgba(255, 100, 68, 0.16) 50.52%,
+                        rgba(255, 68, 79, 0.4) 100%
+                    )
+                    #212329,
+            ),
+            'dark': (
+                'mobile':
+                    radial-gradient(
+                        100% 4130.74% at 0% 100%,
+                        rgba(255, 100, 68, 0.4) 0%,
+                        rgba(255, 100, 68, 0.16) 50.52%,
+                        rgba(255, 68, 79, 0.4) 100%
+                    )
+                    #fbdddd,
+                'desktop':
+                    radial-gradient(
+                        100% 4130.74% at 0% 100%,
+                        rgba(255, 100, 68, 0.4) 0%,
+                        rgba(255, 100, 68, 0.16) 50.52%,
+                        rgba(255, 68, 79, 0.4) 100%
+                    )
+                    #fbdddd,
+            ),
+        ),
     ),
-    'usdt': $usdt,
-    'eusdt': $usdt,
-    'tusdt': $usdt,
 );
 
 /// Wallet background color generator
 ///
 /// This mixin will generate a helper classes for wallet background color based on the given map
 ///
 /// @example
-///     .wallet__usd-bg
-///     .wallet__usd-bg--dark
+///     .wallet-header__usd-bg
+///     .wallet-header__usd-bg--dark
 @mixin wallet-bg-color($currencies-map: $wallet-bg-color) {
-    @each $currency, $themes in $currencies-map {
-        @each $theme, $color in $themes {
-            $t: if($theme == 'dark', '--dark', '');
-            .wallet__#{$currency}-bg#{$t} {
-                background: $color;
+    @each $currency, $types in $currencies-map {
+        @each $type, $themes in $types {
+            @each $theme, $colors in $themes {
+                $t: if($theme == 'dark', '--dark', '');
+
+                .wallet-#{$type}__#{to-lower-case($currency)}-bg#{$t} {
+                    background: map-get($colors, 'desktop');
+
+                    @if $currency == 'demo' {
+                        position: relative;
+                        $demo-icon: if($theme == 'dark', 'wallet-demo-bg-dark', 'wallet-demo-bg-light');
+
+                        &:before {
+                            content: '';
+                            display: block;
+                            position: absolute;
+                            left: 0;
+                            top: 0;
+                            width: 100%;
+                            height: 100%;
+                            background-image: url('../../../../public/images/app/wallet/#{$demo-icon}.svg');
```

Comment:
@heorhi-deriv @sergei-deriv I've checked its usage with George but as removing Watermark wasn't in the scope of my card, I thought it would be better to have another task for it. But your opinion is valid and I can remove it before testing all the wallet related components 👍

Reply:
@hamid-deriv In this case please don't delete code from `wallet-header.scss` and leave the images there, we'll change it in the future

Reply:
@sergei-deriv I will keep the one that is used in the wallet-header component. 👍